### ISSUE TITLE
finetune时的问题

### ISSUE THREAD

**Issue by DengyingFu** [OFA-Sys/Chinese-CLIP#274]
![image](https://github.com/OFA-Sys/Chinese-CLIP/assets/122168368/32dd2650-a7ec-4287-83a1-e1406bb739b0)
请问这是什么原因
我的配置如下
![image](https://github.com/OFA-Sys/Chinese-CLIP/assets/122168368/637515d0-ce9f-4ddb-b780-6226db5fbc32)

**Comment by ChesonHuang** [comment]
同遇到过，单机单卡跑的话，可以用下面的命令，感觉是pytorch版本或者参数的问题
`
##### 单机单卡用下面的命令，不要加分布式参数：
python cn_clip/training/main.py \
          --train-data=${train_data} \
	        --val-data=${val_data} \
          --resume=${resume} \
          --logs=${output_base_dir} \
          --name=${name} \
          --save-step-frequency=${save_step_frequency} \
          --save-epoch-frequency=${save_epoch_frequency} \
          --log-interval=${log_interval} \
          ${report_training_batch_acc} \
          --context-length=${context_length} \
          --warmup=${warmup} \
          --batch-size=${batch_size} \
          --valid-batch-size=${valid_batch_size} \
          --valid-step-interval=${valid_step_interval} \
          --valid-epoch-interval=${valid_epoch_interval} \
          --accum-freq=${accum_freq} \
          --lr=${lr} \
          --wd=${wd} \
          --max-epochs=${max_epochs} \
          --vision-model=${vision_model} \
          ${use_augment} \
          --text-model=${text_model}
`

**Comment by xiaoAugenstern** [comment]
export MASTER_ADDR=localhost
我改了这个就可以啦
